v2/common/src/main/java/com/google/cloud/teleport/v2/transforms/ElasticsearchTransforms.java

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright (C) 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.transforms;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.auto.value.AutoValue;
+import com.google.cloud.teleport.v2.transforms.JavascriptTextTransformer.JavascriptRuntime;
+import java.io.IOException;
+import javax.annotation.Nullable;
+import javax.script.ScriptException;
+import org.apache.beam.sdk.io.elasticsearch.ElasticsearchIO;
+import org.apache.beam.sdk.io.elasticsearch.ElasticsearchIO.ConnectionConfiguration;
+import org.apache.beam.sdk.io.elasticsearch.ElasticsearchIO.RetryConfiguration;
+import org.apache.beam.sdk.io.elasticsearch.ElasticsearchIO.Write.FieldValueExtractFn;
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.Validation.Required;
+import org.apache.beam.sdk.transforms.PTransform;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.PDone;
+import org.joda.time.Duration;
+
+/** Contains common transforms for Elasticsearch such as reading and writing. */
+public class ElasticsearchTransforms {
+
+  /** Convert provided long to {@link Duration}. */
+  private static Duration getDuration(Long milliseconds) {
+    return new Duration(milliseconds);
+  }
+
+  /**
+   * The {@link WriteToElasticsearchOptions} class provides the custom execution options passed by
+   * the executor at the command-line.
+   */
+  public interface WriteToElasticsearchOptions extends PipelineOptions {
```

Comment:
Add an example + default values for each of these options where possible.

Reply:
Defaults and examples have been added where applicable.